tests/property/ic_api/certified_data/src/index.ts

```diff
@@ -10,6 +10,16 @@ import {
     update
 } from 'azle';
 
+type CertifiedData = {
+    data: Uint8Array;
+    certificate: [Uint8Array] | [];
+};
+
+const CertifiedData = IDL.Record({
```

Comment:
Please put IDL object definition above the typescript type

Reply:
done